pkg/controllers/resourceapply/resourceapply.go

```diff
@@ -224,3 +227,55 @@ func applyDaemonSet(ctx context.Context, client coreclientv1.Client, recorder re
 	recorder.Event(required, corev1.EventTypeNormal, "Updated successfully", "Resource was successfully updated")
 	return true, nil
 }
+
+func applyPodDisruptionBudget(ctx context.Context, client coreclientv1.Client, recorder record.EventRecorder, requiredOriginal *policyv1.PodDisruptionBudget) (bool, error) {
```

Comment:
Has this been copied from library go?

Reply:
Aligned it with library-go version, removed spec hash and generation annotation